source/contracts/reporting/ReputationToken.sol

```diff
@@ -108,6 +116,36 @@ contract ReputationToken is DelegationTarget, Extractable, ITyped, Initializable
         return universe;
     }
 
+    function getTotalMigrated() public view returns (uint256) {
+        return totalMigrated;
+    }
+
+    function updateSiblingMigrationTotal(IReputationToken _token) public returns (bool) {
+        require(_token != this);
+        IUniverse _supposedUniverse = _token.getUniverse();
```

Comment:
Any particular reason you use `supposed` rather than `shady` here, in contrast to the usage of `shady` elsewhere for the same concept?  I'm not opposed to `supposed` over `shady`, though I do value consistency.

Reply:
Not really, that's a good point. Will change in my current PR